python/cudf/cudf/utils/gpu_utils.py

```diff
@@ -55,6 +55,10 @@ def validate_setup():
             raise e
         # If there is no GPU detected, set `gpus_count` to -1
         gpus_count = -1
+    except RuntimeError as e:
+        # could indicate that `libcuda.so` is missing
+        warnings.warn(str(e))
```

Comment:
Should we include additional information here in the warning? Like "you probably aren't running on a CUDA capable device?"